integrations/streamer_test.go

```diff
@@ -1323,6 +1324,43 @@ stream
 	testStreamerWithOutput(t, "TestStream_Window_FillPeriod_Aligned", script, 21*time.Second, er, false, nil)
 }
 
+func TestStream_Aggregate_Changing_Type(t *testing.T) {
+
+	var script = `
+var period = 10s
+var every = 10s
+stream
+	|from()
+		.database('dbname')
+		.retentionPolicy('rpname')
+		.measurement('m')
+	|window()
+		.period(period)
+		.every(every)
+	|where(lambda: "c")
+	|count('value')
+	|httpOut('TestStream_Aggregate_Changing_Type')
+`
+
+	er := models.Result{
+		Series: models.Rows{
+			{
+				Name:    "m",
+				Tags:    nil,
+				Columns: []string{"time", "count"},
+				Values: [][]interface{}{
+					{
+						time.Date(1971, 1, 1, 0, 0, 20, 0, time.UTC),
+						1.0,
```

Comment:
To be more clear, the result here will always be an int since count returns an int. That isn't the type that is changing. Rather the type of the aggregate function used to perform the count (yes it is typed) wasn't changing. So in this test the first window is empty and so we choose a data type of float, but on the second window when there are actual values that are ints, we needed to change the aggregator function to be the int type function. Previous to this fix it would have logged an error about the types and returned not data for the count operation.